chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -226,4 +226,11 @@ muu.email.subject.failure=Meaningful Use Upload Failed
 muu.email.body.success=<h3>Meaningful Use Upload Job Summary</h3><p>%s meaningful use records were successfully uploaded.</p>
 muu.email.body.failure=<h3>Meaningful Use Upload Job Failure</h3><p>%s</p>
 
-
+#Cures Statistics report
+curesStatisticsReport.subject=Cures Update Report
+curesStatisticsReport.listingCriterionStatistics.fileName=listings-with-criteria-report-%s-
+curesStatisticsReport.originalCriterionUpgraded.fileName=criteria-upgraded-to-cures-report-%s-
+curesStatisticsReport.curesUpgradedWithoutOriginal.fileName=criteria-upgraded-without-original-report-%s-
+curesStatisticsReport.listingCriterionForCuresAchievement.fileName=listings-to-criteria-needed-for-cures-%s-
+curesStatisticsReport.listingCuresStatusStatistics.emailBody=<h4>Listings With Cures Status (%s)</h4>%s
+curesStatisticsReport.privacyAndSecurityStatistics.emailBody=<h4>Privacy And Security Status (%s)</h4>%s
```

Comment:
Does this need a newline at the end? I just see the red "error" indicator thing and wonder